packages/story-editor/src/components/videoTrim/provider.js

```diff
@@ -56,15 +56,15 @@ function VideoTrimProvider({ children }) {
       return;
     }
     const lengthInSeconds = Math.round(endOffset / 1000 - startOffset / 1000);
+
     trimExistingVideo({
       resource: {
         ...resource,
         length: lengthInSeconds,
         lengthFormatted: getVideoLengthDisplay(lengthInSeconds),
       },
       // This is the ID of the resource that's currently on canvas and needs to be cloned.
-      // It's only different from the above resource, if the canvas resource is a trim of the other.
-      canvasResourceId: element.resource.id,
```

Comment:
Why does this need to be removed?

Reply:
As far as I can see it's not used anywhere else correct me if I'm wrong but it's only used as a parameter for calling `trimExistingVideo `

Which I updated to 
```
const trimExistingVideo = useCallback(
    ({ resource: oldResource, elementId, start, end }) => {
 ```

Reply:
It is used here https://github.com/GoogleForCreators/web-stories-wp/blob/e00e50da7a02d956b993f4038ca1c15d46eda3ef/packages/story-editor/src/app/media/utils/useProcessMedia.js#L211

I don't love the idea of elemnt and resource having the same id. It doesn't make it a unique identifier anymore and makes this code confusing and hard to follow.

Reply:
That ID could theoretically be derived from the element.

As discussed yesterday I am going through this PR in detail. Haven't gotten around to do that yet, but it's high up in my todo list.

Let's hold off on reviewing anything until then.